### DISCUSSION TITLE
Not functioning under the latest update whilst also having start11 installed

### DISCUSSION THREAD

**Discussion by jasonbrianmerrill** [valinet/ExplorerPatcher#438] [General]
Since installing the latest beta of windows, 22000.348, I cannot get explorerpatcher to initiate. I have uninstalled, reinstalled, restarted, restarted file explorer, etc.

one confounding factor could be that I have start11 installed.

Am I doing something wrong? It was working just yesterday. 

thank you. :) 

I apologize if this isn't the right place for this kind of discussion.

**Comment by valinet** [comment]
Yeah, it’s been posted yesterday as well. Microsoft lately does not make symbols for beta/release preview builds available, for whatever reason, so it doesn’t work with those.
Support can be added manually, as I have done for the past 2 beta builds, but it’s just robotic work I don’t want to do anymore. 
I recommend sticking with public releases or builds that have symbols available, or patch in the addresses yourself. The project could use such a contribution :D
These builds anyway do not bring anything useful or new at all, if you ask me, so I honestly cannot understand why so many people bother with them. Compare the number of things implemented in ExplorerPatcher in the past month with things implemented in 348 vs 318, the current public build. And I assure you, I don’t have 20 billion dollars in my account to dispose of.
No matter how you look at it, it’s Microsoft’s fault on all fronts:
* failure to deliver a proper user interface
* failure to upload symbols for public builds, as they should do
* failure to timely improve on things solicited by huge amounts of users, if at all

Thanks

**Reply by Mely2k1** [reply]
So if we accidentally upgrade to the latest beta build this ain't gonna work until the next public build becomes available then?

A shame since this program worked great before.

**Reply by jasonbrianmerrill** [reply]
> *

I hope my post wasn't taken as a critique, I just thought it was peculiar. I am most certainly not capable of anything technical here, so I apologize if this forum is specifically for contributors. I am glad it's been seen already, and I have no problem rolling back. 

And yes, the blame is ENTIRELY on MS. Not you. You're doing a great thing.

thank YOU

**Reply by valinet** [reply]
No, it’s not for contributors only, but this issue has been raised multiple times and some people, not you, live under the impression that I am responsible for this, or that it’s some inherent problem or issue with the program that I just don’t care about fixing or something like that. Like, look at @Mely2k1’s reply above, it gives the feeling that this software fucks up one’s workflow when a new beta Windows build comes “accidentally” (no one puts you in Insider Preview “accidentally”, but whatever), when in fact all it tries to do is solve some issues, not create others. Getting rid of the need for symbols entirely is kind of impossible, but the issue can be alleviated with certain contributions, as I simply cannot take care of everything on my own, I am counting on the community to provide some stuff as well, especially for the things I am not a user of. I don’t run beta builds, so now, what should I do, spend time every 2 weeks doing the robotic work of identifying where things are in the disassembly for each new version? Because it is time consuming and robotic. If someone else wants to do it, fine, if not, it stays like this and that’s it, it’s also more than fine by me. I am also okay with showing anyone how I do it, and if they want to provide this service for each new build, sure, I for one simply can’t do it. I mean, not much to show, you take a file from a build with known symbols, and try to identify some patterns which are likely to be the same in the old file, in order to help you locate where the same function is in the build without symbols. It’s the type of work the human brain is pretty good at, even though the work is boring and repetitive after the 2nd time you do it, and the type of work all the “machine learning” and “artificial intelligence” and all the computers in the world are too stupid to do it manually, or rather, they won’t ever be able to do without breaking more than a sweat because they are not designed to think, but to execute an exact algorithm which there is a lack of in this case. But no one seems to be interested in that, people limit themselves to feeling it’s “a shame” this doesn’t work. And I was just mentioning, the shame is on Microsoft for delivering this cluster fuck, not me or anyone else involved that is doing their best to provide a solution.

But yeah, do not worry, it was not a critique at you or anyone else in general, so apologize to you and anyone if I was or am too harsh.

**Comment by jasonbrianmerrill** [comment]
I am unfamiliar with this platform or I'd search it a bit better. You have every reason to be upset. I wish I knew more about how to work on this kind of thing, but I am a mere audio engineer/musician. 

It's not like the build of windows brings anything new at all.

They have taken away features that made, for me, win10 great. 

Why the taskbar changes? I have to wonder what they were thinking. But you doing this has enabled me to upgrade to windows 11. 

I appreciate you and respect and support your decision and work. You are doing a near-thankless SERVICE. thanks again.

**Reply by valinet** [reply]
Thank you very much and good luck with your work as well!
